packages/composite-checkout/src/components/checkout-provider.js

```diff
@@ -101,7 +101,7 @@ export const CheckoutProvider = ( props ) => {
 	);
 
 	// This error message cannot be translated because translation hasn't loaded yet.
-	const errorMessage = 'Sorry, there was an error loading this page';
+	const errorMessage = 'Sorry, there was an error loading this page.';
```

Comment:
The comment about translations here is no longer true. I missed it when I was updating the translations to use `__`, but maybe it's worth fixing in this PR! See other files in this packages for examples of how to get and use `__`.

Reply:
Thanks for letting me know. I saw it, but had left it b/c of the comment. I'll fix it up.